roles/ceph-osd/templates/ceph-osd-run.sh.j2

```diff
@@ -54,6 +54,12 @@ expose_partitions "$1"
   {% else -%}
   --cpu-quota={{ ceph_osd_docker_cpu_limit * 100000 }} \
   {% endif -%}
+  {% if ceph_osd_docker_cpuset_cpus -%}
```

Comment:
if `is defined` other the current check mean `ceph_osd_docker_cpuset_cpus` is True.